activemq-client/src/main/java/org/apache/activemq/command/ActiveMQTextMessage.java

```diff
@@ -210,4 +210,22 @@ public String toString() {
         }
         return super.toString();
     }
+
+    @SuppressWarnings("unchecked")
+    public boolean isBodyAssignableTo(Class c) throws JMSException {
+        /*
+         *If the message is a TextMessage, ObjectMessage, MapMessage or BytesMessage
+         * and the message has no body, then the above does not apply and this method
+         *  will return true irrespective of the value of this parameter.
+         */
```

Comment:
Could probably make the comment make more sense by not using the full spec sentence :)